### DISCUSSION TITLE
Blazor/FluentUI project setup  for Messaging and interactivity  Need guidance

### DISCUSSION THREAD

**Discussion by vaidhykumar** [microsoft/fluentui-blazor#1604] [Q&A]
I am seeking clarity on server/client separation to get the interactivity aspects to work. I created a project based on template (Fluent Blazor Web App ) and the message bar examples don't work.  In another thread, there was a discussion about moving the examples to a client project ? Is it still necessary in Feb 2024 based on many changes to the underlying framework?

**Comment by vnbaaij** [comment]
Hi, this doesn't seem to be a library related question, but more a general Blazor question...When using both interactiveserver and interactivewebassembly in on solution, you need to put the components you want to use in wasm in a client project yes.

**Comment by vaidhykumar** [comment]
My issues went away after adding the appropriate rendermode  e.g  in the MainLayout.razor
 <NotificationCenter @rendermode="RenderMode.InteractiveServer" />
